R/prep.R

```diff
@@ -220,7 +227,7 @@ getLineAndFile <- function(call){
       endColumn = srcref[4],
       srcbody = srcbody,
       isFile = isFile
-  )}, error=function(e) NULL, silent=TRUE)
+  )}, error=function(e) NULL, silent=getOption('vsc.trySilent', default=TRUE))
```

Comment:
Unfortunately, `silent` will never work here since `tryCatch` does not have a `slient` argumnt like `try`. `...` only accepts `error` to capture error signal by `stop`, `warning` to capture `warning` signal, `message` to capture `message` signal, and `condition` to capture all uncaptured signal conditions.